### PR TITLE
A few minor ReadMe improvements (formatting & link texts)

### PR DESCRIPTION
**Pull request by katrinleinweber**
See diff & https://webaccess.berkeley.edu/ask-pecan/click-here ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,27 +9,27 @@ rOpenSci's guide for packages in our suite. [Read it here](https://ropensci.gith
...
+This book contains our guidelines for packages contributed to [the rOpenSci suite of packages](https://ropensci.org/packages/). They are always a work in progress - corrections, suggestions and general improvements are welcome as [issue submissions in this repository](https://github.com/ropensci/dev_guide/issues/new). Open discussions are welcome in our [forum](https://discuss.ropensci.org/). You can also suggest changes by editing the `.Rmd` files that are at the root of this repository and submitting a pull request.  An "edit" button at the top of all book chapters will take you directly to the relevant page on GitHub to make such changes. Please target your pull requests to the `dev` branch.
```
Comment (maelle) [inline README.md]: what's the change here?

File: README.md [hunk 2]
```diff
@@ -9,27 +9,27 @@ rOpenSci's guide for packages in our suite. [Read it here](https://ropensci.gith
...
 
-If you're an associate editor and you want to render the book locally you need to install `bookdown` and the other dependencies stated in [DESCRIPTION](DESCRIPTION) in particular use `devtools::install_github("bergant/airtabler")`, and get and store an Airtable API key following [there instructions](https://github.com/bergant/airtabler#get-and-store-the-api-key) (if you're not an editor, you don't have access to our Airtable base). Then use `bookdown::render_book('index.Rmd', 'bookdown::gitbook')` and the book will be generated in an _output folder.
+If you're an associate editor and you want to render the book locally you need to install `bookdown` and the other dependencies stated in [DESCRIPTION](DESCRIPTION) in particular use `devtools::install_github("bergant/airtabler")`, and [get and store an Airtable API key following their instructions](https://github.com/bergant/airtabler#get-and-store-the-api-key) (if you're not an editor, you don't have access to our Airtable base). Then use `bookdown::render_book('index.Rmd', 'bookdown::gitbook')` and the book will be generated in an `_output` folder.
```
Comment (maelle) [inline README.md]: the diff isn't clear here either

**Comment by maelle** [comment]
Thanks!

**Comment by katrinleinweber** [comment]
Does this Kaleidoscope diff make the changes clearer?

image

**Comment by maelle** [comment]
Yesss thank you!

**Comment by katrinleinweber** [comment]
You're welcome, and thanks for merging :-)

**Pull request merged**